### ISSUE TITLE
Ruby: Add detail to Rails Assets Pipeline documentation.

### ISSUE THREAD

**Issue by sophiewigmore** [paketo-buildpacks/paketo-website#498]
Recently, we [got feedback](https://github.com/paketo-buildpacks/rails-assets/issues/144#issuecomment-1126934722) that the configuration options in the Rails Assets buildpack are difficult to figure out, and require some hacking to get working.

We should add some additional details to the  Ruby  How-To and Reference documentation about the configuration options and also what the buildpack expects the app to have. Check out the full thread at https://github.com/paketo-buildpacks/rails-assets/issues/144 for some examples.

### Checklist
- [X] I have included links to page(s) that need to be changed

**Comment by kingdonb** [comment]
Just one thing to add:

https://paketo.io/docs/reference/ruby-reference/#rails-asset-pipeline - you shared this link in the other thread, I see now that it has lots of the information I needed to learn about, but it was buried enough that I did not find it on my own perusing the docs.

It is more complicated than I thought to add a "Getting Started" for Ruby

Now that I am taking a sort of broader inventory of the docs format, not sure exactly how to make this better? Not all Ruby apps need Rails, many won't require a Postgres database, but you still sort of need to cover all of this, and by the time you're into databases, the question may come up "should we show how to deploy a postgres database on the cluster" now I've just more than blown up the complexity budget for an effective doc, (it would no longer fit well in the list of examples here):

<img width="513" alt="Screen Shot 2022-05-19 at 2 22 32 PM" src="https://user-images.githubusercontent.com/3286998/169372845-554546d5-2480-4222-a247-93be85ca91bb.png">

At the same time, those are basically "table stakes" for Rails devs and many will walk away if they come and find your docs don't appear to cover all of that.

**Comment by sophiewigmore** [comment]
@kingdonb again, thank you for this feedback, it is majorly helpful! I'm really glad you took the time to walk us through your experience. From what I'm hearing there are a few actionable things from your experience we could take a look at improving:

- Adding some `kpack` documentation to the website seems to be important. A how-to guide could definitely help. 
- Adding more specific/detailed how-to guides to the Ruby docs (for things like connecting to a DB) or at least including that it can be accomplished with the buildpacks and adding links to Ruby docs for realistic use cases might help?
- Add more samples. We have a [samples repository](https://github.com/paketo-buildpacks/samples) with examples, but they're all documented using `pack` and the rails example could go a bit deeper.
- In our top level docs, we could probably make builder selection clearer
- Make all the different docs locations easier to find at a top-level


Separately, I've never used Flux but I'd be interested in seeing a demo with the buildpacks so I can get a better idea of what this workflow looks like. Would you be interested in doing a quick Zoom call or something to show us? Alternatively, we have a[ weekly working group meeting](https://github.com/paketo-buildpacks/community#paketo-project-working-group-meeting) where we could chat about your use case more if you have questions or wanted to show us? Whatever works.

**Comment by kingdonb** [comment]
Yeah, I'm definitely interested in joining the working group meeting. I had found the kpack working group meeting on their calendar already myself, but hadn't seen this one. I am generally available on Tuesdays at 2, so maybe I can join this week!

It just so happens I recorded a 15 minute demo that includes kpack and flux last weekend; it needed to have narration added and I am pretty green at video editing, but I gave it an attempt this morning. The first 7 minutes or so are standard kubeadm setup stuff, then I start to get to the kpack+flux parts... where an `ImageRepository` monitors for new app `builds` (tags) and notifies the automation when they are pushed to my registry by kpack.

Then the flux `ImageUpdateAutomation` resource, through an `ImagePolicy`, selects the latest one to deploy and updates a Deployment manifest with the new image tag. Finally Flux deploys our build and we can see it running.

I am currently rendering it out with OpenShot, I am crossing my fingers that the audio is going to be in sync, as I really don't know what I'm doing here, I'm pretty sure they were originally out of sync because of a difference in sample rates between my two recordings (I could use some help when it comes to video processing, this stuff isn't easy!)

(Sixteen minutes later...) I'm going to upload what I have, here welcome to check this out:

https://youtu.be/CZ1yvFQYiog
